src/Expression/ForClasses/HaveNameEndsWith.php

```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types=1);
+
+namespace Arkitect\Expression\ForClasses;
+
+use Arkitect\Analyzer\ClassDescription;
+use Arkitect\Analyzer\FullyQualifiedClassName;
+use Arkitect\Expression\Description;
+use Arkitect\Expression\Expression;
+use Arkitect\Expression\PositiveDescription;
+use Arkitect\Rules\Violation;
+use Arkitect\Rules\Violations;
+
+class HaveNameEndsWith implements Expression
```

Comment:
English is not my mother language,  but let me ask the same.  Isn't `HaveNameEndingWith` more correct than `HaveNameEndsWith`?